lib/licenses.nix

```diff
@@ -50,6 +50,11 @@ lib.mapAttrs (n: v: v // { shortName = n; }) rec {
     free = false;
   };
 
+  apache20 = spdx {
```

Comment:
This license is already in the file as `asl20`.

Reply:
sorry. missed that.